### PR TITLE
README: ✍️ added Table of Contents

### PR DESCRIPTION
**Pull request by nitinmewar**

### CODE REVIEW

**Comment by renaynay** [comment]
Thank you so much for the contribution @nitinmewar ! 

Is there any way you can split the two commits contained in this PR into separate PRs? One for returning an error with cel-key and the other for updating table of contents?

File: cmd/cel-key/node_types.go [hunk 1]
```diff
@@ -38,7 +39,8 @@ func DirectoryFlags() *flag.FlagSet {
 func ParseDirectoryFlags(cmd *cobra.Command) error {
 	nodeType := cmd.Flag(nodeDirKey).Value.String()
 	if nodeType == "" {
-		return nil
+		err := errors.New("no node type provided")
+		return err
```
Comment (renaynay) [inline cmd/cel-key/node_types.go:43]:
```suggestion
		return errors.New("no node type provided")
```

**Review by renaynay — CHANGES_REQUESTED** [review]
Thank you!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# Table of Content
```
Comment (renaynay) [inline README.md:1]:
```suggestion
# Table of Contents
```

**Review by renaynay — CHANGES_REQUESTED** [review]

**Comment by renaynay** [comment]
@nitinmewar if you implement the suggestion on #1081 , we will merge that first and rebase this PR on top once you've implemented the changes requested here.

**Comment by renaynay** [comment]
@nitinmewar can you rebase this on main?

**Comment by nitinmewar** [comment]
@renaynay yes sure!!

**Comment by nitinmewar** [comment]
@renaynay I am trying to rebase my toe branch to main but it is saying branch is up to date. what am I missing?

**Comment by renaynay** [comment]
git rebase -i upstream/main

make sure this repo is set as upstream

**Review by renaynay — DISMISSED** [review]

**Review by renaynay — COMMENTED** [review]
there is a lint issue @nitinmewar

**Comment by nitinmewar** [comment]
@renaynay what should I do now?

**Comment by renaynay** [comment]
from lint error: 
```
README.md:37:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:38:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:39:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:58:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:59:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:60:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:78:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:79:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:80:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
Error: Process completed with exit code 1.
```

**Comment by Wondertan** [comment]
@nitinmewar, try running ` markdownlint --config .markdownlint.yaml '**/*.md' --fix`, if it does not work, you should fix markdown lint errors manually

**Comment by Wondertan** [comment]
Still failing

**Comment by Wondertan** [comment]
Yes please @nitinmewar

**Review by renaynay — APPROVED** [review]

**Review by Wondertan — APPROVED** [review]
Thanks, @nitinmewar for starting this and @renaynay, for finishing this

**Pull request merged**